pkg/service/test/util.go

```diff
@@ -10,10 +11,37 @@ import (
 	"strings"
 	"time"
 
+	"go.keploy.io/server/pkg/hooks"
 	"go.keploy.io/server/pkg/models"
+	"go.keploy.io/server/pkg/platform"
+	"go.keploy.io/server/pkg/platform/yaml"
+	"go.keploy.io/server/pkg/proxy"
 	"go.uber.org/zap"
 )
 
+type InitialiseRunTestSetReturn struct {
+	Tcs           []*models.TestCase
+	ErrChan       chan error
+	TestReport    *models.TestReport
+	DIDE          bool
+	UserIP        string
+	InitialStatus models.TestRunStatus
+	TcsMocks      []*models.Mock
+}
+
+type InitialiseTestReturn struct {
+	SessionsMap              map[string]string
+	TestReportFS             *yaml.TestReport
+	Ctx                      context.Context
+	AbortStopHooksForcefully bool
+	Ps                       *proxy.ProxySet
+	ExitCmd                  chan bool
+	Ys                       platform.TestCaseDB
```

Comment:
Here since the fields are exported, you can use descriptive name or add comment for the field

Reply:
Change ps, ys and dIDE